octobot_trading/enums.py

```diff
@@ -103,6 +107,7 @@ class PositionStatus(enum.Enum):
     LIQUIDATED = "liquidated"
     OPEN = "open"
     ADL = "auto_deleveraging"
+    CLOSED = "closed"
```

Comment:
Are use sure a position `closed` status exists? If I remember well there is no method to fetch close positions on exchanges. There is only positions history (snapshots of positions that have gone from a size != 0 to 0)

Reply:
Moreover if I remember well, empty (size=0) positions are cosidered as open so I don't see where we could have "close" positions